src/components/Icon/Icon.mdx

```diff
@@ -0,0 +1,48 @@
+---
+name: Icons
+menu: Components
+route: /icons
+---
+
+import { Playground, Props } from 'docz'
+import { ArrowDownCircle, GitHub, Minimize, Minimize2, Figma, LifeBuoy, XCircle, X } from '../../index.ts'
+import Icon from './Icon.tsx'
+
+# Icons
+
+The full set of icons from [Feather Icons](https://feathericons.com).
+
+All icons have their own component. Each component name is a pascal cased version of the [Feather Icon](https://feathericons.com).
+
+## Basic usage
+
+<Playground>
+  <ArrowDownCircle />
+  <GitHub />
+  <Minimize />
+  <Minimize2 />
+  <Figma />
+  <LifeBuoy />
+  <XCircle />
+  <X />
```

Comment:
X is confusing to me. I don't know this component will be an icon from the name. Can we make it `<Icon.X />` ?

Reply:
No, we are directly exporting feather-icons. So we have 0 control over the components. That is also the reason I put this in the docs because it is the only unclear one

Reply:
We have control over the naming. Something like this might work:

```
// index.ts
export { default as Icon } from './components/Icon/Icon';
```

```
// Icon.ts
import * as featherIcons from 'react-feather';
const Icon = featherIcons;
export default Icon;
```

Reply:
This was initially done. But removed to maintain the tree shaking functionality.